<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWidget</class>
 <widget class="QWidget" name="PlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>1121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browsePushButton">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>300</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>1201</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>114</y>
     <width>1201</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="rulerLayout_">
    <item>
     <widget class="QSlider" name="seekSlider">
      <property name="statusTip">
       <string>seekSlider</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>1201</width>
     <height>261</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>tableWidget</string>
   </property>
   <column>
    <property name="text">
     <string>Path</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>1201</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="scrollBarLayout"/>
  </widget>
  <widget class="QSlider" name="vZoomSlider">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>120</y>
     <width>22</width>
     <height>171</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="vZoomSlider"/>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1241</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="16">
     <widget class="QPushButton" name="volPushButton">
      <property name="text">
       <string>vol</string>
      </property>
     </widget>
    </item>
    <item row="0" column="11">
     <widget class="QPushButton" name="revPlayPushButton">
      <property name="toolTip">
       <string>revPlayPushButton</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_pos">
      <property name="text">
       <string>pos</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_beg">
      <property name="text">
       <string>beg</string>
      </property>
     </widget>
    </item>
    <item row="0" column="17">
     <widget class="QComboBox" name="devicesComboBox"/>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_sel">
      <property name="text">
       <string>end</string>
      </property>
     </widget>
    </item>
    <item row="0" column="15">
     <widget class="QPushButton" name="stopPushButton">
      <property name="toolTip">
       <string>stopPushButton</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="18">
     <widget class="QComboBox" name="sampleratesComboBox"/>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_end">
      <property name="text">
       <string>sel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_dur">
      <property name="text">
       <string>dur</string>
      </property>
     </widget>
    </item>
    <item row="0" column="14">
     <widget class="QPushButton" name="pausePushButton">
      <property name="toolTip">
       <string>pausePushButton</string>
      </property>
      <property name="text">
       <string>II</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QPushButton" name="repeatPushButton">
      <property name="toolTip">
       <string>repeatPushButton</string>
      </property>
      <property name="text">
       <string>L</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="10">
     <widget class="QPushButton" name="fastRevPushButton">
      <property name="toolTip">
       <string>fastRevPushButton</string>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="18">
     <widget class="QSlider" name="hZoomSlider">
      <property name="toolTip">
       <string>hZoomSlider</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="12">
     <widget class="QPushButton" name="playPushButton">
      <property name="toolTip">
       <string>playPushButton</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="13">
     <widget class="QPushButton" name="fastPlayPushButton">
      <property name="toolTip">
       <string>fastPlayPushButton</string>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QPushButton" name="cursorIbeamButton">
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>cursorIbeamButton</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string comment="1">I</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">cursorGroupBox</string>
      </attribute>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QPushButton" name="cursorOpenHandButton">
      <property name="toolTip">
       <string>cursorOpenHandButton</string>
      </property>
      <property name="text">
       <string comment="2">H</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">cursorGroupBox</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="12">
     <widget class="QPushButton" name="getInfoPushButton">
      <property name="toolTip">
       <string>Print the Information about the Audio File</string>
      </property>
      <property name="text">
       <string>get Info</string>
      </property>
     </widget>
    </item>
    <item row="1" column="11">
     <widget class="QPushButton" name="testPushButton">
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
    </item>
    <item row="1" column="10">
     <widget class="QPushButton" name="loadWavePushButton">
      <property name="toolTip">
       <string>loadWavePushButton</string>
      </property>
      <property name="text">
       <string>wave</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QPushButton" name="removePushButton">
      <property name="toolTip">
       <string>removePushButton</string>
      </property>
      <property name="text">
       <string>Rem</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QPushButton" name="addAndPlayPushButton">
      <property name="toolTip">
       <string>addAndPlayPushButton</string>
      </property>
      <property name="text">
       <string>ad +</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="addPushButton">
      <property name="toolTip">
       <string>addPushButton</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="cursorArrowButton">
      <property name="toolTip">
       <string>cursorArrowButton</string>
      </property>
      <property name="text">
       <string>A</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">cursorGroupBox</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>330</y>
     <width>31</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="popLayout">
    <item>
     <widget class="QFrame" name="volPopup">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QSlider" name="volumeSlider">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>19</width>
         <height>171</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="cursorGroupBox"/>
 </buttongroups>
</ui>
